CavalcanteLucas/labdale
# Feature/OL44-edit_todo

**Link to task:** 
https://labcodes.atlassian.net/browse/OL-44

**How to test:**
With a user logged in, create a TodoList, then create a Todo. After that, try to edit the Todo's title or the Todo's deadline in any order. See if a dismissable notification of success appear on the DashboardPage when operation is complete.

**Description of your solution:**
This PR implements the backend API and the frontend interfaces that enable Todos to be individually updated. When updating a Todo, notification of success should appear on the topscreen of the dashboard. New tests introduced by this PR covers PUT requests to Todos.
